src/views/LandingPage.jsx

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable react/prop-types */
 /* eslint-disable react/destructuring-assignment */
+/* eslint-disable class-methods-use-this */
 import React, { Component } from 'react';
```

Comment:
@danprocoder you have a lot of eslint rules disabled, kindly address and treat them.